### ISSUE TITLE
Add note on Checked Out Items page that items without check boxes can't be renewed

### ISSUE THREAD

**Issue by gibsonjc** [CARLI/vufind#273]
NPU - The checked out items page looks rough… it seems that if an item cannot be renewed there is no checkbox there. Would it be possible to add a message to the non-renewable items – something like “cannot be renewed”.

**Comment by dgree1** [comment]
How about text like "Must have box by title to have renewal options." 
![checkboxes_helptext_ex](https://user-images.githubusercontent.com/29785703/36399018-9360a64a-158f-11e8-873a-592d02bd66cd.GIF)

**Comment by dgree1** [comment]
Or something simpler like 
"No box, no option to renewal."

**Comment by dmcmpbll** [comment]
We asked the CARLI Instruction Committee to weigh in on possible wording and suggestions for placement. Their suggested wording is:

"Items without checkboxes are not eligible for renewal."

Their suggested placement is either directly below, or directly above the space with the renewal buttons, marked "B" and "C" in this screenshot. Some preference was given to "C" over "B", if both locations were possible.

![image](https://user-images.githubusercontent.com/20848151/37306197-f1069ac2-2604-11e8-9af5-675b01278ebc.png)

**Comment by gibsonjc** [comment]
On March 7, 2018, we implemented a different API to obtain Voyager data which presents the renew checkboxes on the New VuFind Your Checked Out Items page for all items at all times even when the item is not renewable, so this wording is no longer necessary.  If  user checks the box and tries to renew an item that cannot be renewed, it will present the message: "This item could not be renewed. Not renewed."  VuFind 0.6 works this same way.  Closing this Issue.
